maps/southern_cross/job/outfits_vr.dm

```diff
@@ -0,0 +1,17 @@
+/decl/hierarchy/outfit/job/pathfinder
+	name = OUTFIT_JOB_NAME("Pathfinder")
+	shoes = /obj/item/clothing/shoes/boots/winter/explorer
+	uniform = /obj/item/clothing/under/explorer //TODO: Uniforms.
+	l_ear = /obj/item/device/radio/headset/explorer
+	id_slot = slot_wear_id
+	pda_slot = slot_l_store
+	pda_type = /obj/item/device/pda/heads
```

Comment:
This PDA comes with the ability to set status displays and other stuff. They are ***not a head job***.